config.example.toml

```diff
@@ -424,6 +424,11 @@
 # What custom diff tool to use for displaying compiletest tests.
 #compiletest-diff-tool = <none>
 
+# Indicates whether ccache is used when building certain artifacts (e.g. LLVM).
+# Set to `true` to use the first `ccache` in PATH, or set an absolute path to use
+# a specific version.
+#ccache = false
+
```

Comment:
Should we not remove `llvm.ccache` ? We have 2 `ccache` fields documented now.

Reply:
I wanted to keep the option in example before we fully remove llvm.ccache. Do you want me to remove it?